site/assets/scss/_content.scss

```diff
@@ -5,20 +5,11 @@
 .bd-content {
   order: 1;
 
-  // Hack the sticky header
-  // stylelint-disable selector-no-qualifying-type
-  > h2[id],
-  > h3[id],
-  > h4[id] {
-    pointer-events: none;
-
-    &::before {
-      display: block;
-      height: 6rem;
-      margin-top: -6rem;
-      content: "";
+  // Offset for the sticky header
+  > * {
```

Comment:
I think we could still limit this to headings, no?

Reply:
I've now limited it to elements with an id, just in case someone links to another element.